test/regression/convert/generate-manifests.go

```diff
@@ -1,7 +1,20 @@
+// generate-manifests.go
+//
+// Renders registry+v1 bundles into YAML manifests for regression testing.
+// Used by tests to make sure output from the BundleRenderer stays consistent.
+//
+// By default, writes to ./testdata/tmp/generate/.
```

Comment:
You say “By default, writes to ./testdata/tmp/generate/” but the test uses `./testdata/tmp/rendered/`. Either unify on one directory name or drop the default entirely and require the flag, so there’s no confusion.